content/2_Bronze/Time_Comp.mdx

```diff
@@ -451,3 +452,74 @@ triplets.
 <!-- Bitsets are significantly faster (and so are bitwise operations vs iterating through bits). I don't want to make it sound insignificant -->
 
 For now, don't worry about optimizing constant factors -- just be aware of them.
+
+## Quiz
+<Quiz>
+  <Quiz.Question>
+    What's time complexity?
+    <Quiz.Answer>
+      Measuring the amount of memory an algorithm consumes.
+      <Quiz.Explanation>
+        This is known as space complexity.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Measuring the number of operations an algorithm performs.
+      <Quiz.Explanation>
+        You'd always want your algorithm to run as fast as possible!
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      The time it takes you to solve a problem.
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What time complexity would a mathematical formula take?

```

Comment:
I mean this isn't true for all mathematical formulas (for example fast expo). Maybe be more specific (what time complexity would addition take).